internal/core/models/integration_models.go

```diff
@@ -49,4 +49,5 @@ type GetIntegrationQueryResult struct {
 	AutoPiDefaultTemplateID      int                    `json:"auto_pi_default_template_id"`
 	RefreshLimitSecs             int                    `json:"refresh_limit_secs"`
 	AutoPiPowertrainToTemplateID map[PowertrainType]int `json:"auto_pi_power_train_to_template_id,omitempty"`
+	TokenID                      int                    `json:"token_id"`
```

Comment:
What populates this? Is there some reflection going on? How easy is it to run this locally?

Reply:
Like I think there are `get_in_*.go` and associated tests here that probably have to be dealt with:

https://github.com/DIMO-Network/device-definitions-api/tree/40d1c99991bec422a39252e3045e76c5f3d89c64/internal/core/queries